### ISSUE TITLE
dynpro checks: CHAIN indentation false positive

### ISSUE THREAD

**Issue by mbtools** [abaplint/abaplint#3511]
`CHAIN.` block should be indented

```abap
CHAIN. 
  FIELD input3. 
  FIELD input4. 
  FIELD input5. 
  MODULE check_chain ON CHAIN-REQUEST. 
ENDCHAIN. 
```

https://help.sap.com/doc/abapdocu_latest_index_htm/latest/en-US/index.htm?file=dynpchain.htm

**Comment by larshp** [comment]
unable to reproduce, what is the full example?

**Comment by larshp** [comment]
screen example added in playground

**Comment by mbtools** [comment]
```abap
PROCESS BEFORE OUTPUT.
  MODULE pbo_1001.

PROCESS AFTER INPUT.
  CHAIN.
    FIELD test.
  ENDCHAIN.
```

```
/zfoo.prog.screen_0100.abap[6, 5]: Indentation problem, expected 2 spaces(indentation)
```

**Comment by larshp** [comment]
@mbtools try again, 2.113.85 published

**Comment by mbtools** [comment]
Thank you, it works... in other repos. 

![image](https://github.com/user-attachments/assets/fd5e30c6-247d-45db-aff6-d13e0d219844)

On https://github.com/Marc-Bernard-Tools/MBT-System-Monitor, I tried new PRs and commits but it is stuck on 2.113.78. 🤷 How do I get it to 2.113.85?

**Comment by larshp** [comment]
@mbtools try again

**Comment by mbtools** [comment]
almost. it now runs 2.113.85 but getting "Main file must have specific contents: Report must begin with REPORT or PROGRAM"

the error is also there if the main program is included in the commit.

![image](https://github.com/user-attachments/assets/0253e583-7f04-4303-914d-cc8354d7dd20)

**Comment by larshp** [comment]
@mbtools try again
